tests/models/test_semantic_segmentation.py

```diff
@@ -55,6 +55,11 @@ def _rm_dir(path):
         shutil.rmtree(path, ignore_errors=True)
 
 
+def _crash_func(x):
+    """Helper to induce crash."""
+    raise ValueError("Propagation Crash.")
```

Comment:
Please can elaborate this error a bit more?

Reply:
Its an induced error to simulate an exception occured when the code is running. 

https://github.com/TIA-Lab/tiatoolbox/blob/30488ba15a8d7fa0caa30f0ea11b23e24de446f8/tiatoolbox/models/controller/semantic_segmentor.py#L898-L912

Depending on how we set the flag here, we either crash outright with a full stack trace printed out, or we will dump out the stack trace within a log file and carry on with the next input WSI/tile.

In either cases, it would be easier to test the above this if we can induce some specific crash pattern and exceptions. For example. I tested for crash on exception here
https://github.com/TIA-Lab/tiatoolbox/blob/948b85bf31edd009b54175e847de081884f2a865/tests/models/test_semantic_segmentation.py#L321-L332
The func above is helper to generate the pattern. Its can be of any exception type though, I only use ValueError here for conveniences.